ci-operator/step-registry/ipi/conf/aws/oidc-creds-provision/ipi-conf-aws-oidc-creds-provision-commands.sh

```diff
@@ -22,6 +22,28 @@ echo "RELEASE_IMAGE_LATEST_FROM_BUILD_FARM: ${RELEASE_IMAGE_LATEST_FROM_BUILD_FA
 oc registry login
 oc adm release extract --credentials-requests --cloud=aws --to="/tmp/credrequests" "${RELEASE_IMAGE_LATEST_FROM_BUILD_FARM}"
 
+if [[ ${ENABLE_SHARED_VPC} == "yes" ]]; then
+  ingress_cr_file="/tmp/credrequests/0000_50_cluster-ingress-operator_00-ingress-credentials-request.yaml"
```

Comment:
Shall we add one more check the file exists or not?

Reply:
Updated